packages/expo-checkbox/unimodule.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "expo-checkbox",
+  "platforms": ["android", "web"]
```

```suggestion
  "platforms": ["web"]
```

to avoid autolinking